src/astro/files/type/__init__.py

```diff
@@ -0,0 +1,39 @@
+import pathlib
+from typing import Dict, Type, Union
+
+from astro.constants import FileType as FileTypeConstants
+from astro.files.type.base import FileType
+from astro.files.type.csv import CSV
+from astro.files.type.json import Json
+from astro.files.type.ndjson import NdJson
+from astro.files.type.parquet import Parquet
+
+
+def create_type_factory(path: str):
```

Comment:
It would be great if we rename this to `create_file_type`, so it is consistent with FileLocation and Database.

Reply:
Done.